src/app/settings/lights/lights.component.html

```diff
@@ -63,19 +61,22 @@
   </ion-note>
 </ion-item>
 <ion-item>
-          <ion-range label="LED Maximum Brightness [{{rescueConf.lightMaxBrightness}}]" min="0" max="255" step="1" [(ngModel)]="rescueConf.lightMaxBrightness" pin snaps color="secondary">
+          <ion-range label="LED Maximum Brightness [{{rescueConf.lightMaxBrightness}}]" labelPlacement="stacked" min="0" max="255" 
+                     step="1" [(ngModel)]="rescueConf.lightMaxBrightness" pin snaps color="secondary">
             <ion-icon slot="start" size="small" color="secondary" name="sunny"></ion-icon>
```

Comment:
**Suggestion:** It's better to use a more descriptive label for the LED Maximum Brightness range.
```suggestion
          <ion-range label="LED Maximum Brightness (Current: {{rescueConf.lightMaxBrightness}})" labelPlacement="stacked" min="0" max="255" 
                     step="1" [(ngModel)]="rescueConf.lightMaxBrightness" pin snaps color="secondary">
          </ion-range>
```